src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx

```diff
@@ -40,138 +40,141 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
   /etc/newrelic-infra/integrations.d
   ```
 
-2. Use the below snippet to update your configuration file named `sendmail-flex-config.yml`.
+2. Update your configuration file `sendmail-flex-config.yml` using the following snippet:
+
+Kindly update the below value present in the snippet:
+`FAILED_MESSAGES_FILE_NAME`: Go to the path `/var/mail/` and check the file that has been created for failed messages
```

Comment:
```suggestion
2. Update your configuration file `sendmail-flex-config.yml` using the following snippet. 

Make sure to replace "FAILED_MESSAGES_FILE_NAME" references with your file name. To find this file name, go to `/var/mail/` and check the file that has been created for failed messages.
```

@pkosanam How does this look?

Reply:
Thank you @ally-sassman, this looks good to me.. 👍